lib/comic.dart

```diff
@@ -75,7 +79,11 @@ class ComicScreen extends StatelessWidget {
         if (snapshot.connectionState == ConnectionState.waiting) {
           return const Center(child: CircularProgressIndicator());
         } else if (snapshot.hasError) {
-          return Center(child: Text('Error: ${snapshot.error}'));
+          return Center(
+              child: Text(
+            'An error occurred while trying to connect to the server.\nPlease check your connection or try again later.',
+            textAlign: TextAlign.center,
+          ));
```

Comment:
Since the PR is mainly for improving readability, please also split this line:
```suggestion
              child: Text(
            'An error occurred while trying to connect to the server.'
            '\nPlease check your connection or try again later.',
            textAlign: TextAlign.center,
          ));
```

Reply:
Hi @fernando79513, I'm closing this PR because all the changes here have been incorporated into this one: [#19](https://github.com/canonical/standup-timer/pull/19). Thanks!